allauth/socialaccount/providers/fxa/provider.py

```diff
@@ -16,7 +51,10 @@ class FirefoxAccountsProvider(OAuth2Provider):
     account_class = FirefoxAccountsAccount
 
     def get_default_scope(self):
-        return ['profile']
+        if FXA_ON_WHITELIST:
```

Comment:
[It tripped me up as well](https://github.com/mozilla/fxa-oauth-server/issues/143#issuecomment-184819931), prompting me to make this PR. I think this will become better documented later, when (if?) the FxA team prioritizes opening up to 3rd party reliers.